diff --git a/build.gradle b/build.gradle
index b714bf6..33f3872 100644
--- a/build.gradle
+++ b/build.gradle
@@ -16,8 +16,8 @@ dependencies {
     implementation 'it.unimi.dsi:fastutil-core:8.5.16'
     implementation 'commons-codec:commons-codec:1.19.0'
     implementation 'org.bouncycastle:bcprov-jdk18on:1.81'
-    implementation 'com.github.FireMasterK:NewPipeExtractor:a0073ab65fed411480e757e2102356664546307d'
-    implementation 'com.github.FireMasterK:nanojson:a507525e549a836c3a8b6ab7090dca38e92942ef'
+    implementation 'com.github.TeamNewPipe:NewPipeExtractor:v0.26.1'
+    implementation 'com.github.TeamNewPipe:nanojson:c7a6c1c08d16b6d5ecded34758e6415e07be2166'
     implementation 'com.fasterxml.jackson.core:jackson-core:2.19.2'
     implementation 'com.fasterxml.jackson.core:jackson-annotations:2.19.2'
     implementation 'com.fasterxml.jackson.core:jackson-databind:2.19.2'
